bin/post_compile

```diff
@@ -8,12 +8,29 @@ set -euo pipefail
 # Uses the node binaries/packages installed by the nodejs buildpack previously.
 npm install && npm run build
 
+# We compress files in `build/` below so they can be served more quickly
+# but we want to cache the results for pyodide as it changes relatively
+# frequently and compressing it to brotli is sloooooow
+PYODIDE_DIR=$CACHE_DIR/pyodide-$PYODIDE_VERSION
```

Comment:
Where is `CACHE_DIR` defined?  Is that just something that Heroku defines?

Reply:
Yup. https://devcenter.heroku.com/articles/buildpack-api#caching 

I'll put a note about that in the comment